proto/streamlit/proto/DeckGlJsonChart.proto

```diff
@@ -17,11 +17,14 @@
 syntax = "proto3";
 
 message DeckGlJsonChart {
-  // The dataframe that will be used as the chart's main data source.
+  // The json of the pydeck object (https://deckgl.readthedocs.io/en/latest/deck.html)
   string json = 1;
 
   string tooltip = 2;
 
   // If True, will overwrite the chart width spec to fit to container.
   bool use_container_width = 4;
+
+  // the hash of the json so the the frontend doesn't always have to parse the pydeck json object
+  string element_id = 5;
```

Comment:
nit: we have the `id` field which we are using for all the widget. I would suggest to use `id` here as well since the the semantics here are pretty similar:

```suggestion
  string id = 5;
```

Reply:
Ah thanks. I'll do that